categorical_map/categorical_map.ipynb

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
Should we add a markdown cell with the title?

Reply:
done